Pages/homePage.js

```diff
@@ -0,0 +1,51 @@
+class HomePage {
+    constructor() {
+        this.title= "HomePage";
+    }
+    get loginButton (){
+        return browser.driver.findElement(by.className("header-text"));
+    }
+    get fashionCategory(){
+        return browser.driver.findElement(by.css("div.category-item a"));
+    }
+    get logoutButton (){
+        return browser.driver.findElement(by.css("div.login-acccount button"));
+    }
+    get createAnAccount(){
+        return browser.driver.findElement(by.css("a.header-text"));
+    }
+    get aboutUs(){
+        return browser.driver.findElement(by.linkText("About Us"))
+    }
+    get termsAndConditions(){
+        return browser.driver.findElement(by.linkText("Tearms and Conditions"));
+    }
+    get privacyAndPolicy(){
+        return browser.driver.findElement(by.linkText("Privacy and Policy"));
+    }
+    clickOnPrivacyAndPolicy(){
+        return this.privacyAndPolicy.click();
+    }
+    clickOnTermsAndConditions(){
+        return this.termsAndConditions.click();
+    }
+    clickOnAboutUs(){
+        return this.aboutUs.click();
+    }
+    clickOnCreateAnAccont(){
+        return this.createAnAccount.click();
+    }
+    clickOnLogin() {
+       return this.loginButton.click();
+    };
+    clickOnLogoutButton(){
+        this.logoutButton.click()
+    }
+    clickOnFashionCategory(){
+       return this.fashionCategory.click();
+    };
+    open (url){
```

Comment:
Open what ? Can you be more desriptive with method names.
Also, for every getter name try to add part "button", "link" etc

Reply:
I added the description part in console.log(), and also where it had to be, in method name